strawberry/extensions/tracing/opentracing.py

```diff
@@ -0,0 +1,237 @@
+import dataclasses
+import time
+from copy import deepcopy
+from datetime import datetime
+from inspect import isawaitable
+from typing import Any, Callable, Dict, List, Optional
+
+from graphql import GraphQLResolveInfo
+from opentracing import Scope, Tracer, global_tracer
+from opentracing.ext import tags
+from strawberry.extensions import Extension
+from strawberry.types.execution import ExecutionContext
+
+
+DATETIME_FORMAT = "%Y-%m-%dT%H:%M:%S.%fZ"
+
+ArgFilter = Callable[[Dict[str, Any], GraphQLResolveInfo], Dict[str, Any]]
+
+
+def get_path_from_info(info: GraphQLResolveInfo) -> List[str]:
+    path = info.path
+    elements = []
+
+    while path:
+        elements.append(path.key)
+        path = path.prev
+
+    return elements[::-1]
```

Comment:
let's move this to `strawberry.utils.info` (a new file)

Reply:
Yeah! since we're using the same for Apollo too